docs/manual/manual_ifci_control.rst

```diff
@@ -7,23 +7,86 @@
+Like throttle, in order to calculate the raw value that your module will apply to either the X or Y pulsing command, simply divide the received control value by 65535. 
+The raw value is then mapped to the range [-1, 1].
 
-Each control value is two bytes and can encode either a throttle value, an X value, or a Y value. These are considered 'raw values' and represent a range of 0.0 to 1.0. The following sections contain more information about how the modules can use these values. A module can read any individual control value in the Packed Control Message through a corresponding Control Value Index (CVI) value. These CVIs are set in the :ref:`IQUART Flight Controller Interface Client<ifci_message_table>`. For example, if throttle_cvi is set to 2 and a packed control message has a control value in index 2 then the module will set its throttle based on the value at that control index. The same is true for x_cvi and y_cvi. The module's responses to control values are defined by settings in the :ref:`ESC Propeller Input Parser<epip_message_table>` and :ref:`Pulsing Rectangular Input Parser<prip_message_table>`.
+Suppose your module receives an X control value of 42000. The equivalent raw value is :math:`\frac{42000}{65535} = 0.641`. In order to map that value onto [-1, 1], simply multiply it by 
+two and subtract one. In this case :math:`(0.641 * 2) - 1 = +0.284`.
 
-More information on how throttle control values will be interpreted can be found in :ref:`manual_throttle`.
+Your module also receives a Y control value of 1000. The raw value is :math:`\frac{1000}{65535} = 0.015`, and the Y control percentage is :math:`(0.015*2) - 1 = -0.97`.
 
-The X and Y CVIs convert the raw 0.0 to 1.0 values to -1.0 to 1.0. In the :ref:`Pulsing Rectangular Input Parser<prip_message_table>`, the setting 'pulsing_voltage_mode' controls whether this value is considered a percentage of supply voltage or a percentage of the value defined by 'pulsing_voltage_limit'. When set to supply voltage mode -1.0 maps to a pulsing voltage on that axis of negative battery voltage and 1.0 maps to positive battery voltage. When set to voltage limit mode you must also set the 'pulsing_voltage_limit' parameter. This will set the maximum and minimum pulsing voltage. In this mode -1.0 will map to -'pulsing_voltage_limit' and 1.0 will map to 'pulsing_voltage_limit'. Between these values, the raw value is linearly mapped to the voltages in between. For example if the battery voltage is 12V and 'pulsing_voltage_mode' is set to 0 then -1.0 would map to -12V of pulsing, 0.0 would map to 0V, and 1.0 would map to 12V. With the same battery voltage, but 'pulsing_voltage_mode' set to 1, and 'pulsing_voltage_limit' set to 4.0, -1.0 would map to -4V of pulsing, 0.0 would map to 0V and 1.0 would map to 4V of pulsing.
+In the Pulsing Rectangular Input Parser, the setting ``pulsing_voltage_mode`` controls whether this value is considered a percentage of supply voltage or a percentage of the 
```

Comment:
Maybe throw a link to the pulsing rectangular input parser table here when you mention it?